static_src/stores/login_store.js

```diff
@@ -7,25 +7,44 @@ import BaseStore from './base_store.js';
 import { loginActionTypes } from '../constants.js';
 
 
-class LoginStore extends BaseStore {
+export class LoginStore extends BaseStore {
   constructor() {
     super();
     this.subscribe(() => this._registerToActions.bind(this));
+    // TODO this should probably be false, but we need to account for the
```

Comment:
out of scope for this PR, but we should really look into using something like enums to get achieve something close to a tri state boolean.